src/test/components/UrlManager.test.js

```diff
@@ -328,4 +329,47 @@ describe('UrlManager', function () {
     // timeline track organization state.
     expect(getTimelineTrackOrganization(getState()).type).toBe('active-tab');
   });
+
+  it('can handle a from-post-message data source', async () => {
```

Comment:
Thanks for the test ! <3

Reply:
I don't want anyone breaking my shit! :D